### DISCUSSION TITLE
set log level with environment variable

### DISCUSSION THREAD

**Discussion by zhaoyang-intel** [microsoft/onnxruntime#8112] [Other Q&A] [answered]
Hi,

I know there are ways to set log level programmatically in the c++ api.

Is there a way log level can be changed by setting environment variable (eg, export ORT_LOG_LEVEL="INFO")?

**Comment by edgchen1** [comment, marked as answer]
Not that I'm aware of. I think this would be useful though - will create an issue.

**Reply by zhaoyang-intel** [reply]
Thank you!
